readme.md

```diff
@@ -8,9 +8,9 @@ This is a C/C++ code analysis plugin for Eclipse that shows bugs detected by the
 
 * Linux operating system
 * Recent [CodeChecker](https://github.com/Ericsson/codechecker) (6.8.0 and up)
-* Every eclipse version since [Oxygen 1](https://www.eclipse.org/downloads/packages/release/oxygen/1) (v4.7.0) is supported
+* Every eclipse version since [Phtoton](https://www.eclipse.org/downloads/packages/release/photon/r) (v4.8.0) is supported
```

Comment:
I think we should split up the documentation to be more explicit what are the plugin build requirements and what is the runtime requirement for it.

This change mainly effects the build environment if I'm right.

Reply:
True that! 
But for the current setup of the compilation, we only test for one target platform, so until we create a deeper matrix with separate, older (newer) target platforms, we can't be sure the plugin works on older releases, even though we won't use newer features. I'll create an issue regarding this problem.
Until then, I can extend the readme with a note that it should work on Oxygen.